libjob_queue/src/forward_model.c

```diff
@@ -256,9 +258,10 @@ void forward_model_formatted_fprintf(const forward_model_type * forward_model ,
                                      const char * path,
                                      const char * data_root,
                                      const subst_list_type * global_args,
-                                     mode_t umask) {
+                                     mode_t umask,
+                                     env_varlist_type * list) {
   forward_model_python_fprintf( forward_model, path, global_args, umask);
```

Comment:
When you are in these parts of the code - just remove the `forward_model_python_fprintf` function completely.

Reply:
That branches out to several other problems. This commit is just about the container. I suggest removing forward_model_python_fprintf as part of the complete ERT-1375 task.